terraform/groups/ecs-service/locals.tf

```diff
@@ -8,8 +8,8 @@ locals {
   docker_repo                 = "psc-verification-api"
   kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority   = 160
-  lb_listener_paths           = ["/*"]
-  healthcheck_path            = "/"
+  lb_listener_paths           = ["/persons-with-significant-control-verification*"]
```

Comment:
do we need a forward slash at the end? `/persons-with-significant-control-verification/*`

Reply:
I don't believe we need that, I'm going with what's in the routes file